admin/users.sql

```diff
@@ -124,3 +124,9 @@ CREATE USER IF NOT EXISTS AD_SERVICE
 CREATE USER IF NOT EXISTS RECOVER_SERVICE
     PASSWORD = 'placeholder'
     MUST_CHANGE_PASSWORD = TRUE;
+
+CREATE USER IF NOT EXISTS ADMIN_SERVICE
+    TYPE = SERVICE;
+
+CREATE USER IF NOT EXISTS DEVELOPER_SERVICE
```

Comment:
Are we no longer going to use the `DPE_SERVICE` service account?

Reply:
See my comment [here](https://sagebionetworks.jira.com/browse/SNOW-89?focusedCommentId=237696). I thought that `DEVELOPER_SERVICE` was a more specific title for what this user is intended to do.